rust/lon/src/cli.rs

```diff
@@ -486,34 +486,53 @@ fn bot_fallible(directory: impl AsRef<Path>, forge: &impl Forge, base_ref: &str)
     Ok(())
 }
 
-struct CommitMessage(Vec<(String, UpdateSummary)>);
+struct CommitMessage {
+    updates: Vec<(String, UpdateSummary)>,
+    flatten: bool,
+}
 
 impl CommitMessage {
-    fn new() -> Self {
-        Self(vec![])
+    fn new(flatten: bool) -> Self {
+        Self {
+            updates: vec![],
+            flatten,
+        }
     }
 
     fn add_summary(&mut self, name: &str, summary: UpdateSummary) {
-        self.0.push((name.into(), summary));
+        self.updates.push((name.into(), summary));
     }
 
     fn is_empty(&self) -> bool {
-        self.0.is_empty()
+        self.updates.is_empty()
     }
 }
 
 impl fmt::Display for CommitMessage {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         let mut commit_message = String::new();
-        writeln!(&mut commit_message, "lon: update")?;
-        writeln!(&mut commit_message)?;
-        writeln!(&mut commit_message, "Updated sources:")?;
-        writeln!(&mut commit_message)?;
-
-        for (name, summary) in &self.0 {
-            writeln!(&mut commit_message, "• {name}:")?;
-            writeln!(&mut commit_message, "    {}", summary.old_revision)?;
-            writeln!(&mut commit_message, "  → {}", summary.new_revision)?;
+
+        if self.flatten && self.updates.len() == 1 {
```

Comment:
I think we should not add a flatten option but whenever `self.updates.len() == 1` just include the name it the tagline. This would also satisfy your use case, right?